bitcoin/base58.c

```diff
@@ -67,40 +67,24 @@ static bool from_base58(u8 *version,
 	return true;
 }
 
-bool bitcoin_from_base58(bool *test_net,
-			 struct bitcoin_address *addr,
+bool bitcoin_from_base58(u8 *version, struct bitcoin_address *addr,
```

Comment:
is it worth preserving this method (and the ones below) if they're just a wrapper for another?

Reply:
I was tempted to remove them, however the type-safety that this method gives (`struct bitcoin_address` wraps `struct ripemd160`) is worth it I think.